pyproject.toml

```diff
@@ -71,8 +72,8 @@ exclude = '''
 )
 '''
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry_core>=1.0.0"]
```

Comment:
What does this do? I see this in other places, but would be good to understand.

Reply:
That was the original config poetry used, they have now updated their docs with this config (this should be changed elsewhere)

Reply:
I think I read this wrong; this is just specifying the version of poetry that should be used to do poetry things. Per the docs, these reqs are PEP517 compliant

https://python-poetry.org/docs/pyproject/#poetry-and-pep-517